Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyOriginatorKeyFile>FluentAssertions.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>1591;1573</NoWarn>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
Using a .NET 5 compiler introduces a warning about EOL Frameworks such as netcoreapp2.0 and netcoreapp3.0